src/api/app/controllers/webui/driver_update_controller.rb

```diff
@@ -107,7 +107,7 @@ def save
   # TODO: select architecture of binary packages
   def binaries
     required_parameters :repository
-    @binaries = @package.build_result(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
+    @binaries = @package.build_result_old(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
```

Comment:
Ok, my main concern is that nobody will end up removing this code if we don't do it now.. But a trello card as long we do get it done soon is fine for me as well.